content/pl/build-release-run.md

```diff
@@ -0,0 +1,18 @@
+## V. Buduj, wypuszczaj, uruchamiaj
+### Precyzyjnie oddziel etap budowania od uruchamiania
+
+[Codebase](./codebase) jest przetwarzany w (nie developerskie) wdrożenie w trzech etapach
+
+* Podczas *etapu budowania* kod z repozytorium jest konwertowany do wykonywalnej paczki tzw. *buildu*. Używając wersji kodu zdefiniowanej przez commit w procesie deploymentu, w etapie budowania aplikacji pobiera się, dołącza do projektu [zależności](./dependencies) i kompiluje niezbędne zasoby.
+* W *etapie wypuszczania* aplikacji brany jest build stworzony w poprzednim etapie i konfigurowany na bazie [ustawień](./config) obecnego wdrożenia.  Stworzony w ten sposób *release* zawiera zbudowane źródło kodu, jego konfigurację i jest gotowe do uruchomienia w wybranym środowisku.
+* *Etap uruchamiania* (znany również jako "runtime") odpala aplikację w środowisku wykonawczym przez uruchomienie zestawu [procesów](./processes) opisanych w przygotowanym releasie.
+
+![Kod staje się buildem, jeśli zostanie połączony z konfiguracją by stworzyć release](/images/release.png)
+
+**Aplikacja dwunastu aspektów ścieśle rozgranicza etapy budowy, wydania i uruchomienia**  Nie jest więc możliwe zrobić zmian kodu podczas działania aplikacji, ponieważ nie można przenieść ich z powrotem do etapu budowy.
```

Comment:
1. `Nie jest więc możliwe zrobić zmian kodu podczas działania aplikacji` -> mocno nie po polsku to jest :)